src/components/Router.js

```diff
@@ -20,12 +27,27 @@ class Router extends Component {
     }
   }
 
+  getAuthenticatedUser() {
+    Auth.currentAuthenticatedUser()
+      .then(user => {
+        const email = user.attributes.email
+        if (email.substring(email.indexOf("@") + 1, email.length) === 'ubcbiztech.com') {
```

Comment:
I'm assuming we will be doing backend verification as well? 
https://softwareengineering.stackexchange.com/questions/53262/do-you-use-both-client-side-and-server-side-validation-techniques

Reply:
Not sure how to modify Auth backend. 

When we implement account creation, we can do backend verification there (and set the user to an admin in the backend, instead of checking in the front end)

Reply:
I think doing both will be good

Reply:
Both as in front end + backend?

Reply:
indeed, as suggested my the link i sent